src/flags.rs

```diff
@@ -8,6 +8,7 @@ pub struct Flags {
     pub display_tree: bool,
     pub display_indicators: bool,
     pub recursive: bool,
+    pub sort: (SortFlag, SortOrder),
```

Comment:
I think you should split this tuple into two variable as it is two different flags and if you pass all the struct to the `sort()` method you would have no need of it.